website/sidebars.js

```diff
@@ -49,6 +49,7 @@ const sidebarSettings = {
           items: [
             "docs/cloud/about-cloud-setup",
             "docs/dbt-cloud-environments",
+            "docs/cloud/account-settings",
```

Comment:
This might be better served placed above the Environments page

Reply:
great idea! lemme move it